Automattic/woocommerce-payments
# Disputes: Allow merchant to respond to inquiries from transaction detail page

Fixes #7193

#### Changes proposed in this Pull Request

Add handling of CTA and modal for when a dispute is of type Inquiry.

This PR mostly focuses on minor wording changes.

Inquiries have separate actions that can be undertaken as compared with a normal dispute. They can be challenged but not accepted. With Inquiries the `accept` action has been replaced with a `refund` action. 

When clicking the primary action `Submit Evidence` button, the user is taken through the normal flow to submit evidence for the dispute/inquiry. As with disputes, the secondary action button `Issue refund` pops up a modal with more information.

The primary action button inside the modal `View Order to Issue Refund`. It needs to be very clear to the merchant that they must issue a refund manually.  

An automatic refund ability may be added later, likely once #7248 is implemented.

#### Text changes

| | Dispute | Inquiry |
|--------|--------|--------|
| CTA Buttons | `Challenge Dispute` / `Accept Dispute` | `Submit Evidence` / `Issue Refund` |
| Modal Title | `Accept the dispute?` | `Issue a refund?` |
| Modal Content | Accepting the dispute marks it as Lost. The disputed amount will be returned to the cardholder, with a $15.00 USD dispute fee deducted from your account.<br /><br />Accepting the dispute is final and cannot be undone. | Issuing a refund will close the inquiry, returning the amount in question back to the cardholder. No additional fees apply.<br /><br />This action is final and cannot be undone.<br /><br />You will be taken to the order, where you must complete the refund process manually.. |
| Modal Buttons | `Cancel` / `Accept Dispute` | `Cancel` / `View Order to Issue Refund` | 

#### Screenshots

| Dispute | Inquiry |
|--------|--------|
| ![Screenshot 2023-09-27 at 1 46 44 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/825998cd-2fd4-45a2-9a87-770f7b813b9b) | ![Screenshot 2023-09-27 at 1 46 51 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/213f0e5c-bfdd-4eeb-9b97-2fc636c1479c) |
| ![Screenshot 2023-09-27 at 1 47 05 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/f5d644e5-e89b-411a-89f1-0f9867856cd3) | ![Screenshot 2023-09-27 at 1 46 59 PM](https://github.com/Automattic/woocommerce-payments/assets/57298/1311917a-d124-4575-a323-9b4926981b5d) |

#### Testing instructions

1. Enable feature flag by running `update_option( '_wcpay_feature_dispute_on_transaction_page', '1' );` in [WP Console](https://wordpress.org/plugins/wp-console/) or by modifying your database table `wp_options` directly.
2. Create a disputed  / inquiry transaction using the test card `4000000000001976` at checkout.
3. Visit the transaction details screen, it should show the `Submit Evidence` / `Issue Refund` buttons.

-------------------


**Post merge**
